deploy/olm-catalog/openshift-pipelines-operator/openshift-pipelines-operator.package.yaml

```diff
@@ -1,4 +1,5 @@
+---
```

Comment:
could you remove this line. it is making the configmap generated using `scripts/olm_catalog.sh` invalid.

Reply:
I will handle this and push a patch with OLM related changes